packages/ladle/lib/cli/vite-plugin/generate/cleanup-windows-path.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @param {string} path
+ * @return {string}
+ */
+function cleanupWindowsPath(path) {
+  return path.replace(/\\/g, "/");
```

Comment:
That reverses slashes though, maybe something like this?

```suggestion
  return path.replace(/\\/g, "\\\\");
```

Reply:
doesn't `/` work everywhere in Windows these days (NT6+)?

I always use it in powershell and node scripts and stuff.

I also checked `cmd.exe` and it had no issues with:
- `cd "Documents/My Web Sites/"`
- running an executable: `C:\>"Program Files (x86)/Steam/steamapps/common/Crusader Kings III/binaries/ck3.exe"`

It would seem that, while under the hood it still uses `\` as it's canonical directory path separator, [Windows converts `/` at the Windows API level](https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats#path-normalization), lower than anything in userspace.

Reply:
@wojtekmaj I think your suggestion can also work, but I think generating the same code independent of the OS is better in this case no?

Otherwise you risk more problems down the road, how the generated code is used.

`/` is supported in windows as imports in typescript.

Reply:
Yeah, you're both right, I was being probably a bit too conservative with this one.